<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQLyapWidget.ui,v $ </comment>
<comment>   $Revision: 1.4 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/09/16 12:57:40 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQLyapWidget</class>
 <widget class="TaskWidget" name="CQLyapWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>127</height>
   </rect>
  </property>
  <property name="caption" stdset="0" >
   <string>CQLyapWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="mpCheckDelay" >
         <property name="text" >
          <string>start averaging after t =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mpEditDelay" />
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mpEditExponent" />
     </item>
     <item row="2" column="1" >
      <widget class="QCheckBox" name="mpCheckDivergence" >
       <property name="text" >
        <string>calculate Divergence</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLblExponents" >
       <property name="text" >
        <string>Number of Exponents</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="mpLine" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>TaskWidget</class>
   <extends>QWidget</extends>
   <header>TaskWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditExponent</tabstop>
  <tabstop>mpCheckDelay</tabstop>
  <tabstop>mpEditDelay</tabstop>
  <tabstop>mpCheckDivergence</tabstop>
 </tabstops>
 <includes>
  <include location="local" >TaskWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpCheckDelay</sender>
   <signal>clicked()</signal>
   <receiver>CQLyapWidget</receiver>
   <slot>slotDelayChecked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
